### PR TITLE
{Release} Upgrade to Azure CLI 2.19.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,137 @@
...
+
+**Key Vault**
+
+* `az keyvault key import`: Support `--curve` parameter for importing BYOK keys (#16593)
+* `az keyvault certificate download`: Fix deprecated/removed method call (#16319)
+* `az keyvault create/update`: Remove preview tag for `--enable-rbac-authorization` (#16630)
+
+**Monitor**
+
+* `az monitor metrics alert create`: fix resource is not found error (#16741)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:77]:
```suggestion
* `az monitor metrics alert create`: Fix 'resource is not found' error (#16741)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,137 @@
...
+* Add managed cluster stop/start bindings (#16599)
+* `az aks check-acr`: Fix Kubernetes version check (#16718)
+
+**APIM**
+
+* GA the command group (#16811)
+
+**App Config**
+
+* [BREAKING CHANGE] az appconfig feature filter add: (#16536)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:25]:
```suggestion
* [BREAKING CHANGE] `az appconfig feature filter add`: Support adding JSON objects as feature filter parameter values (#16536)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,137 @@
...
+* `az network application-gateway  client-cert`: Manage trusted client certificate of application gateway (#16762)
+* `az network application-gateway  ssl-profile`:  Manage ssl profiles of application gateway (#16762)
+
+**Profile**
+
+* `az login`: Launch browser in WSL 2 (#16556)
+
+**RDBMS**
+
+* : `az mysql flexible-server create --iops <value>` : Allow user to choose IOPS for their SKU. (#15831)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:100]:
```suggestion
*`az mysql flexible-server create --iops` : Allow user to choose IOPS for their SKU. (#15831)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,137 @@
...
+
+**BotService**
+
+* `az bot create/update`: Adding Encryption features --cmk-key-url and --encryption-off (#16694)
+* `az bot update`: Renaming Encryption-OFF arg to CMK-OFF and updating api version (#16794)
+
+**Compute**
+
+* [BREAKING CHANGE] vmss create: rename orchestration mode values (#16726)
+* New command group sshkey (#16331)
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst:54]:
```suggestion
* New command group sshkey. Allow referencing a SSH key resource when creating a VM (#16331)
```

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,137 @@
...
+
+**App Config**
+
+* [BREAKING CHANGE] az appconfig feature filter add: (#16536)
+
+**App Service**
+
+* az appservice ase/plan ASEv3 support (#16516)
+* Fix #16026 and #16118 for az appservice plan (#16516)
+* Add support for os-preference to fix #16509 (#16575)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:31]:
```suggestion
* Fix #16509: Add support for os-preference (#16575)
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,137 @@
...
+
+**Compute**
+
+* [BREAKING CHANGE] vmss create: rename orchestration mode values (#16726)
+* New command group sshkey. Allow referencing a SSH key resource when creating a VM (#16331)
+* `az disk create/update`: Add parameter `--enable-bursting` to support disk bursting (#16702)
+
+**DigitalTwins**
+
+* Adds support for private endpoint connections to DigitalTwins (#16668)
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst:59]:
```suggestion
* Add support for private endpoint connections to DigitalTwins (#16668)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,137 @@
...
+
+**App Service**
+
+* az appservice ase/plan ASEv3 support (#16516)
+* Fix #16026 and #16118 for az appservice plan (#16516)
+* Add support for os-preference to fix #16509 (#16575)
+* Improve behavior of appservice ase create-inbound-services to allow skipping DNS services and support DNS for ASEv2 (#16575)
+* `az webapp up/az webapp create`: Fix nonetype errors (#16605)
+* `az webapp up/create`: better error handling of app name with period (#16623)
+* Fix #16681: az webapp config ssl import: fixing bug that causes failures on national clouds (#16701)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:35]:
```suggestion
* Fix #16681: `az webapp config ssl import`: fixing bug that causes failures on national clouds (#16701)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,137 @@
...
+* `az provider register`: Support registering management group (#15847)
+
+**Backup**
+
+* Add CRR functionality for IaaSVM and other CRR commands (#16557)
+* `az backup protectable-item list`: Add protectable-item-type as an optional argument (#16765)
+
+**BotService**
+
+* `az bot create/update`: Adding Encryption features --cmk-key-url and --encryption-off (#16694)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:48]:
```suggestion
* `az bot create/update`: Add Encryption features `--cmk-key-url` and `--encryption-off` (#16694)
```

File: src/azure-cli/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,137 @@
...
+* `az bot update`: Renaming Encryption-OFF arg to CMK-OFF and updating api version (#16794)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:49]:
```suggestion
* `az bot update`: Rename Encryption-OFF arg to CMK-OFF and updating api version (#16794)
```

File: src/azure-cli/HISTORY.rst [hunk 10]
```diff
@@ -3,6 +3,137 @@
...
+
+**Compute**
+
+* [BREAKING CHANGE] vmss create: rename orchestration mode values (#16726)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:53]:
```suggestion
* [BREAKING CHANGE] vmss create: Rename orchestration mode values (#16726)
```

File: src/azure-cli/HISTORY.rst [hunk 11]
```diff
@@ -3,6 +3,137 @@
...
+* New command group sshkey (#16331)
+* `az disk create/update`: Add parameter `--enable-bursting` to support disk bursting (#16702)
+
+**DigitalTwins**
+
+* Adds support for private endpoint connections to DigitalTwins (#16668)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:59]:
```suggestion
* Add support for private endpoint connections to DigitalTwins (#16668)
```

File: src/azure-cli/HISTORY.rst [hunk 12]
```diff
@@ -3,6 +3,137 @@
...
+* `az keyvault certificate download`: Fix deprecated/removed method call (#16319)
+* `az keyvault create/update`: Remove preview tag for `--enable-rbac-authorization` (#16630)
+
+**Monitor**
+
+* `az monitor metrics alert create`: Fix 'resource is not found' error (#16741)
+
+**NetAppFiles**
+
+* `az netappfiles account ad add`: parameter security-operators added. (#16467)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:81]:
```suggestion
* `az netappfiles account ad add`: Add parameter `--security-operators`. (#16467)
```

File: src/azure-cli/HISTORY.rst [hunk 13]
```diff
@@ -3,6 +3,137 @@
...
+* `az netappfiles volume create`: parameter smb-continuously-available added. (#16467)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:82]:
```suggestion
* `az netappfiles volume create`: Add parameter `smb-continuously-available`. (#16467)
```

File: src/azure-cli/HISTORY.rst [hunk 14]
```diff
@@ -3,6 +3,137 @@
...
+* `az netappfiles volume create`: parameter smb-encryption added. (#16467)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:83]:
```suggestion
* `az netappfiles volume create`: Add parameter `smb-encryption`. (#16467)
```

File: src/azure-cli/HISTORY.rst [hunk 15]
```diff
@@ -3,6 +3,137 @@
...
+* `az netappfiles`: No longer in preview mode. (#16467)
+
+**Network**
+
+* [BREAKING CHANGE] `az network vrouter`: Depracate this command group, please use `az network routeserver`. (#16494)
+* `az network routeserver`: Add new command group. (#16494)
+* `az network application-gateway  create`: Add parameter `--ssl-profile-id` (#16762)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:90]:
```suggestion
* `az network application-gateway create`: Add parameter `--ssl-profile-id` (#16762)
```

File: src/azure-cli/HISTORY.rst [hunk 16]
```diff
@@ -3,6 +3,137 @@
...
+* `az network application-gateway  client-cert`: Manage trusted client certificate of application gateway (#16762)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:91]:
```suggestion
* `az network application-gateway client-cert`: Manage trusted client certificate of application gateway (#16762)
```

File: src/azure-cli/HISTORY.rst [hunk 17]
```diff
@@ -3,6 +3,137 @@
...
+* `az network application-gateway  ssl-profile`:  Manage ssl profiles of application gateway (#16762)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:92]:
```suggestion
* `az network application-gateway ssl-profile`:  Manage ssl profiles of application gateway (#16762)
```

Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:92]:
```suggestion
* `az network application-gateway ssl-profile`: Manage ssl profiles of application gateway (#16762)
```

File: src/azure-cli/HISTORY.rst [hunk 18]
```diff
@@ -3,6 +3,137 @@
...
+* Add new commands for `az security` (#16398)
+
+**SQL**
+
+* Add managed hsm regex match to SQL (#15109)
+* Upgrade azure-mgmt-sql to 0.26.0 (#16618)
+* `az sql mi create/update`: Add support for maintenance configuration in managed instance operations (#16229)
+
+**SQL Server DevOps auditing policy**
+
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst:118]:
```suggestion
```

File: src/azure-cli-core/HISTORY.rst [hunk 19]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.19.0
+++++++
+* New error output (#16257)
```
Comment (houk-ms) [inline src/azure-cli-core/HISTORY.rst:8]:
```suggestion
* New error output: support multiple failure recommendations with colorization (#16257)
```

File: src/azure-cli/HISTORY.rst [hunk 20]
```diff
@@ -3,6 +3,137 @@
...
+* `az netappfiles volume create`: parameter smb-encryption added. (#16467)
+* `az netappfiles`: No longer in preview mode. (#16467)
+
+**Network**
+
+* [BREAKING CHANGE] `az network vrouter`: Depracate this command group, please use `az network routeserver`. (#16494)
+* `az network routeserver`: Add new command group. (#16494)
+* `az network application-gateway  create`: Add parameter `--ssl-profile-id` (#16762)
+* `az network application-gateway  client-cert`: Manage trusted client certificate of application gateway (#16762)
+* `az network application-gateway  ssl-profile`:  Manage ssl profiles of application gateway (#16762)
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:92]:
```suggestion
* `az network application-gateway create`: Add parameter `--ssl-profile-id` (#16762)
* `az network application-gateway client-cert`: Manage trusted client certificate of application gateway (#16762)
* `az network application-gateway ssl-profile`:  Manage ssl profiles of application gateway (#16762)
```

File: src/azure-cli/HISTORY.rst [hunk 21]
```diff
@@ -3,6 +3,137 @@
...
+
+* Add new commands for `az security` (#16398)
+
+**SQL**
+
+* Add managed hsm regex match to SQL (#15109)
+* Upgrade azure-mgmt-sql to 0.26.0 (#16618)
+* `az sql mi create/update`: Add support for maintenance configuration in managed instance operations (#16229)
+
+**SQL Server DevOps auditing policy**
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:117]:
```suggestion
**SQL Server**
```

File: src/azure-cli/HISTORY.rst [hunk 22]
```diff
@@ -3,6 +3,137 @@
...
+* Fix #16489: Upgrade azcopy to 10.8.0 (#16552)
+* `az storage account blob-service-properties`: Support default service version (#16682)
+* Fix #16519: azcopy is given more powerful SAS than needed (has write, only needs read) (#16731)
+
+**Synapse**
+
+* `az synapse workspace create `: Add key-identifier parameter to support to create workspace using customer-managed key. (#16224)
+* `az synapse workspace key`: Add CRUD cmdlets to support to manage keys under specified synapse workspace. (#16224)
+* `az synapse workspace managed-identity`: Add cmdlets to support CRUD managed identity to sql access setting. (#16224)
+* `az synapse workspace`: Add data exfiltration protection support, add allowed_aad_tenant_ids parameter. (#16224)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:135]:
```suggestion
* `az synapse workspace`: Add data exfiltration protection support, add parameter `--allowed-tenant-ids`. (#16224)
```

File: src/azure-cli/HISTORY.rst [hunk 23]
```diff
@@ -3,6 +3,137 @@
...
+
+**SQL**
+
+* Add managed hsm regex match to SQL (#15109)
+* Upgrade azure-mgmt-sql to 0.26.0 (#16618)
+* `az sql mi create/update`: Add support for maintenance configuration in managed instance operations (#16229)
+
+**SQL Server DevOps auditing policy**
+
+* Support SQL server DevOps audit policy commands (#16595)
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:119]:
```suggestion
* `az sql mi create/update`: Add support for maintenance configuration in managed instance operations (#16229)
* Support SQL server DevOps audit policy commands (#16595)
```

File: src/azure-cli/HISTORY.rst (hunk 18 above)
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:118]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 24]
```diff
@@ -3,6 +3,137 @@
...
+**Compute**
+
+* [BREAKING CHANGE] vmss create: rename orchestration mode values (#16726)
+* New command group sshkey. Allow referencing a SSH key resource when creating a VM (#16331)
+* `az disk create/update`: Add parameter `--enable-bursting` to support disk bursting (#16702)
+
+**DigitalTwins**
+
+* Adds support for private endpoint connections to DigitalTwins (#16668)
+
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:60]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 25]
```diff
@@ -3,6 +3,130 @@
...
+* `az netappfiles`: No longer in preview mode. (#16467)
+
+**Network**
+
+* [BREAKING CHANGE] `az network vrouter`: Depracate this command group, please use `az network routeserver`. (#16494)
+* `az network routeserver`: Add new command group. (#16494)
+* `az network application-gateway create`: Add parameter `--ssl-profile-id` (#16762)
+* `az network application-gateway client-cert`: Manage trusted client certificate of application gateway (#16762)
+* `az network application-gateway ssl-profile`: Manage ssl profiles of application gateway (#16762)
+
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:89]:
```suggestion
* Add support for private endpoint connections to DigitalTwins (#16668)

```

File: src/azure-cli/HISTORY.rst [hunk 26]
```diff
@@ -3,6 +3,130 @@
...
+* GA the command group (#16811)
+
+**App Config**
+
+* [BREAKING CHANGE] `az appconfig feature filter add`: Support adding JSON objects as feature filter parameter values (#16536)
+
+**App Service**
+
+* az appservice ase/plan ASEv3 support (#16516)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:29]:
```suggestion
* `az appservice ase/plan`: Support ASEv3 (#16516)
```

File: src/azure-cli/HISTORY.rst [hunk 27]
```diff
@@ -3,6 +3,130 @@
...
+* Fix #16026 and #16118 for az appservice plan (#16516)
+* Fix #16509: Add support for os-preference (#16575)
+* Improve behavior of appservice ase create-inbound-services to allow skipping DNS services and support DNS for ASEv2 (#16575)
+* `az webapp up/az webapp create`: Fix nonetype errors (#16605)
+* `az webapp up/create`: better error handling of app name with period (#16623)
+* Fix #16681: `az webapp config ssl import`: fixing bug that causes failures on national clouds (#16701)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:35]:
```suggestion
* Fix #16681: `az webapp config ssl import`: Fix bug that causes failures on national clouds (#16701)
```

File: src/azure-cli/HISTORY.rst [hunk 28]
```diff
@@ -3,6 +3,130 @@
...
+* Fix #16079: public blob gives error (#16578)
+* GA Storage routing reference (#16550)
+* Fix #9158: Cannot generate a working SAS key from a policy (#16549)
+* Fix #16489: Upgrade azcopy to 10.8.0 (#16552)
+* `az storage account blob-service-properties`: Support default service version (#16682)
+* Fix #16519: azcopy is given more powerful SAS than needed (has write, only needs read) (#16731)
+
+**Synapse**
+
+* `az synapse workspace create `: Add key-identifier parameter to support to create workspace using customer-managed key. (#16224)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:125]:
```suggestion
* `az synapse workspace create `: Add parameter `--key-identifier` to support to create workspace using customer-managed key. (#16224)
```

**Review by zhoxing-ms — APPROVED** [review]

**Review by houk-ms — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by Juliehzl — APPROVED** [review]

**Review by fengzhou-msft — APPROVED** [review]

**Review by qwordy — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**